docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Perseo documentation
+
+Perseo: Documentation
+
```

Comment:
Redundant?

```suggestion
```

Reply:
Looking to https://github.com/telefonicaid/fiware-orion/tree/master/doc/manuals/admin/ it seems to be a kind of pattern... Thus, better to keep it.

NTC